Bio/SeqIO/__init__.py

```diff
@@ -473,11 +475,12 @@
                    "phd": PhdIO.PhdWriter,
                    "qual": QualityIO.QualPhredWriter,
                    "sff": SffIO.SffWriter,
+                   "nib": NibIO.NibWriter,
```

Comment:
This should be a couple of lines higher up (alphabetical), you could fix the ``pir`` entry too at the same time?